db-tools-ij/src/main/grammar/ij.jj

```diff
@@ -1175,6 +1175,29 @@ class ij {
 			verifyTableExists(schema,table);
 
 			DatabaseMetaData dbmd = theConnection.getMetaData();
+
+            //Check if it's a synonym table
+            String getSynonymAliasInfo = "SELECT ALIASINFO \n" +
+                    " FROM \n" +
+                    " SYS.SYSALIASES A, SYS.SYSTABLES T, SYSVW.SYSSCHEMASVIEW S \n" +
+                    " WHERE A.ALIASTYPE LIKE 'S' AND \n" +
+                    " S.SCHEMAID = T.SCHEMAID AND \n" +
+                    " A.ALIAS = T.TABLENAME AND \n" +
```

Comment:
We need to add A.schemaid = T.schemaid to differentiate aliases of same name in different schemas.

Reply:
Low level mistake ...